app/src/main/java/com/onelist/ui/MainActivity.kt

```diff
@@ -215,11 +204,6 @@ class MainActivity : ComponentActivity() {
                     title = { Text(text = "OneList") },
                     backgroundColor = MaterialTheme.colors.primary,
                     contentColor = MaterialTheme.colors.onPrimary,
-//                navigationIcon = {
-//                    IconButton(onClick = {}) {
-//                        Icon(Icons.Filled.ArrowBack, "backIcon")
-//                    }
-//                },
```

Comment:
Good to remove this.

Avoid pushing commented code to Version Control.  Version control gives us better options:
- You can look at history to see what used to be there.
- You can use branches for experimental features.